log/field.go

```diff
@@ -0,0 +1,204 @@
+package log
```

Comment:
Don't feel very strongly. If we don't do it now, it will require a major version bump to add references, assuming we release the current state as 1.0